src/game.cpp

```diff
@@ -4156,7 +4156,7 @@ bool Game::combatChangeHealth(Creature* attacker, Creature* target, CombatDamage
 {
 	const Position& targetPos = target->getPosition();
 	if (damage.primary.value > 0) {
-		if (target->getHealth() <= 0) {
+		if (!target->hasHealth()) {
```

Comment:
no, no, such quality of life improvement should be accompanied by `isDead` function

Reply:
isDead?

Reply:
Every creature has health. The only thing that you can assert on it is whether it is alive (>0) or dead.

Reply:
Thanks!